docs/faq/lukso/project-origins.md

```diff
@@ -0,0 +1,52 @@
+- **More Detailed Transactions**: In ERC777, the 'send' function includes transaction data, not just the amount and recipient. This additional information can be helpful for dApps that need to process more than just simple transfers.
+- **Minting and Burning Events**: ERC777 allows the tracking of token minting and burning events, which can be helpful for dApps that need to monitor the total supply of a token dynamically.
+
+### How did the Genesis Validators start the network?
+
+The [Genesis Deposit Contract](https://etherscan.io/address/0x42000421dd80D1e90E56E87e6eE18D7770b9F8cC#code) was open for about three weeks and was frozen on the 9th of May. The PoS consensus was started with all Genesis Validators that deposited up to block 17,227,300 on Ethereum. There was a 46,523 block delay after the LUKSO team triggered the freeze. The genesis deposit contract received 10.336 deposits from 223 unique addresses based on the [Dune Analytics Dashboard](https://dune.com/hmc/lukso-genesis-validators). Each validator had to deposit 32 LYXe.
+
+Based LUKSO mainnet was started on the [23rd of May of 2023](https://medium.com/lukso/genesis-validators-start-your-clients-fe01db8f3fba) using the genesis files created from the deposits on Ethereum. All future validator deposits will be made as regular validators.
+
+### Where can I find the LUKSO Genesis Deposit Contract?
+
+The official address of the Genesis Deposit Contract is `0x42000421dd80d1e90e56e87e6ee18d7770b9f8cc`. The Code of the contract can be found here. You can also see the list of all valid token transfers until the contract has been closed. We also have a Dune Dashboard for Genesis Validators showing created deposits.
+
+### Which features did the Genesis Deposit Contract have?
+
+The deposit contract called the external LYXe [ERC777](https://ethereum.org/de/developers/docs/standards/tokens/erc-777/) token contract to lock the deposit amounts. To prepare the genesis files of the mainnet launch featured a freeze. Therefore, the deposit contract featured checks if a public key has been registered, preventing duplicate deposits. Only static deposit amounts of 32 LYX were allowed.
+
+The freeze also came with a delay function to allow people to have time still after the LUKSO team triggered it. Each validator that went through the Genesis Launchpad stored not only the LYXe token but also a vote for the initial coin supply of the mainnet. The additional data was stored on-chain to have a publically verifiable source of the initial LYX supply and distribution.
+
+### How much gas did the Genesis Deposit Contract consume?
+
+The Genesis Validator Deposit Contract consumed around `300k` gas per validator key. The contract needed quite some gas for good reasons, as it can be seen as the portal into a new blockchain. Additional features were required to have a safe and verifiable way for LUKSO's genesis.
```

Comment:
this is not relevant, you can remove this. No one will use this contract ever again.

```suggestion
```

Reply:
check